### ISSUE TITLE
changing class_ids from None to{1,2,3,4,5}, results in error while training in coco.py

### ISSUE THREAD

**Issue by xiaolisarah** [matterport/Mask_RCNN#415]
i just choose 5 class to train ,then it is just still,what can i do ?

/home/jgq/anaconda3/envs/python34/lib/python3.4/site-packages/tensorflow/python/ops/gradients_impl.py:95: UserWarning: Converting sparse IndexedSlices to a dense Tensor of unknown shape. This may consume a large amount of memory.
  "Converting sparse IndexedSlices to a dense Tensor of unknown shape. "
/home/jgq/anaconda3/envs/python34/lib/python3.4/site-packages/keras/engine/training.py:1987: UserWarning: Using a generator with `use_multiprocessing=True` and multiple workers may duplicate your data. Please consider using the`keras.utils.Sequence class.
  UserWarning('Using a generator with `use_multiprocessing=True`'
Epoch 1/40

i debug this ,and find the problem maybe in model.py in linenumber 2308-2318
self.keras_model.fit_generator(
                train_generator,
                initial_epoch=self.epoch,
                epochs=epochs,
                steps_per_epoch=self.config.STEPS_PER_EPOCH,
                callbacks=callbacks,
                validation_data=val_generator,
                validation_steps=self.config.VALIDATION_STEPS,
                max_queue_size=100,
                workers=workers,
                use_multiprocessing=True,
here the  validation_data=val_generator,but,it is do nothing ,can u help me? @ waleedka

**Comment by xiaolisarah** [comment]
i just solved the problem

**Comment by xiaolisarah** [comment]
just the loss is nan
